app/views/support_interface/validation_errors/vendor_api/summary.html.erb

```diff
@@ -0,0 +1,97 @@
+                },
+                class: 'sortedby-label',
+                selected: 2,
+                onchange: 'this.form.submit()',
+                role: 'listbox',
+              ) %>
+        </div>
+        <%= f.govuk_submit('Update') %>
+      <% end %>
+    </div>
+  </div>
+</div>
+
+<table class="govuk-table">
+  <thead class="govuk-table__head">
+    <tr class="govuk-table__row">
+      <th class="govuk-table__header"></th>
+      <th class="govuk-table__header"></th>
+      <th class="govuk-table__header govuk-table__header--numeric" colspan="2" width="10%">All time</th>
+      <th class="govuk-table__header govuk-table__header--numeric" colspan="2" width="10%">Last month</th>
+      <th class="govuk-table__header govuk-table__header--numeric" colspan="2" width="10%">Last week</th>
+    </tr>
+    <tr class="govuk-table__row">
+      <th class="govuk-table__header govuk-table__header--numeric"></th>
+      <th class="govuk-table__header govuk-table__header--numeric"></th>
+      <th class="govuk-table__header govuk-table__header--numeric">Error<br>count</th>
+      <th class="govuk-table__header govuk-table__header--numeric">Unique<br>providers</th>
+      <th class="govuk-table__header govuk-table__header--numeric">Error<br>count</th>
+      <th class="govuk-table__header govuk-table__header--numeric">Unique<br>providers</th>
+      <th class="govuk-table__header govuk-table__header--numeric">Error<br>count</th>
+      <th class="govuk-table__header govuk-table__header--numeric">Unique<br>providers</th>
+    </tr>
+  </thead>
+  <tbody class="govuk-table__body">
+  <% @validation_error_summary.each do |row| %>
+    <tr class="govuk-table__row">
+      <td class="govuk-table__cell">
+        <%= govuk_link_to(
+          row['request_path'],
+          support_interface_validation_errors_vendor_api_search_path(request_path: row['request_path']),
+        ) %>
+      </td>
+      <td class="govuk-table__cell">
+        <%= govuk_link_to(
+          row['attribute'],
+          support_interface_validation_errors_vendor_api_search_path(request_path: row['request_path'], attribute: row['attribute']),
+        ) %>
+      </td>
+      <td class="govuk-table__cell govuk-table__cell--numeric">
+        <%= row['incidents_all_time'] %>
+      </td>
+      <td class="govuk-table__cell govuk-table__cell--numeric">
+        <%= row['unique_providers_all_time'] %>
+      </td>
+      <td class="govuk-table__cell govuk-table__cell--numeric">
+        <%= row['incidents_last_month'] %>
+      </td>
+      <td class="govuk-table__cell govuk-table__cell--numeric">
+        <%= row['unique_providers_last_month'] %>
+      </td>
+      <td class="govuk-table__cell govuk-table__cell--numeric">
+        <%= row['incidents_last_week'] %>
+      </td>
+      <td class="govuk-table__cell govuk-table__cell--numeric">
+        <%= row['unique_providers_last_week'] %>
+      </td>
+    </tr>
+  <% end %>
+  </tbody>
+</table>
```

Comment:
Can this be moved in a reusable component (so it can be shared between vendor api and candidate)? 
Consider passing the main body (lines 62 to 95 here) as I can see how it may be difficult to use a shared component for that.

Reply:
done in 1dda5794495a138725549e3e11ff2fbcf7951e14